src/Avalonia.Visuals/Media/TextFormatting/TextFormatterImpl.cs

```diff
@@ -391,6 +391,10 @@ private static bool TryGetLineBreak(TextRun textRun, out LineBreak lineBreak)
                                 measuredLength = currentBreakPosition + lineBreaker.Current.PositionWrap;
                             }
                         }
+                        else if (measuredLength == 0 && currentLength == 0)
+                        {
+                            measuredLength = 1;
+                        }
```

Comment:
@Gillibald pls can you take a look is that fix correct attempt for the #4806, thanks